content/working-with-azure-key-vault-in-csharp/Working with Azure Key Vault in CSharp.md

```diff
@@ -0,0 +1,451 @@
+```
+---
+title: **Working with Azure Key Vault in C\#**
```

Comment:
Remove asterisk from the title, use can use double quotes instead

Reply:
Hi @puneetsingh24 I've made all the changes and checked in the code.